fixup-mountpoints

```diff
@@ -115,6 +115,36 @@ case "$DEVICE" in
             -e 's block/platform/msm_sdcc.1/by-name/LTALabel mmcblk1 ' \
             "$@"
         ;;
+    "z3c" | "d5803" | "aries")
```

Comment:
A note for future: Xiaomi Mi 2/2S has codename "aries", so we'll probably need some CM/AOSP detection